### PR TITLE
mesa: add head

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mesa.rb [hunk 1]
```diff
@@ -4,13 +4,17 @@ class Mesa < Formula
   homepage "https://www.mesa3d.org/"
   url "https://mesa.freedesktop.org/archive/mesa-18.2.3.tar.xz"
   sha256 "e2bf83c17e1abdecb1ee81af22652e27e9aa38f963e95e60f34275cc0376304f"
-
```
Comment (fxcoudert) [inline Formula/mesa.rb]: No whitespace change

File: Formula/mesa.rb [hunk 2]
```diff
@@ -4,13 +4,17 @@ class Mesa < Formula
...
   bottle do
     sha256 "80348f778a9e90548084f7970a633fa26d25eb1b4ca3b298b741ae6c6203bd1a" => :mojave
     sha256 "f050183c77597d5561bbc663d8f0124769005bc6f879e80f7ffcfad8c5f2ad38" => :high_sierra
     sha256 "cafd1cfeced20001aa26800bcb2bbb58339f81ba5744aff52eca6ddf74de5d45" => :sierra
   end
 
+  head do
+    depends_on "gettext"
```
Comment (fxcoudert) [inline Formula/mesa.rb]: Why does it need `gettext`?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/mesa.rb (hunk 1 above)
Comment (yurikoles) [reply Formula/mesa.rb]: Fixed, thanks!

File: Formula/mesa.rb (hunk 2 above)
Comment (yurikoles) [reply Formula/mesa.rb]: Because it fails to build without it.

**Review by fxcoudert — APPROVED** [review]

**Pull request merged**